structuredheadings/plugin.js

```diff
@@ -227,8 +220,9 @@
  */
 
   CKEDITOR.plugins.add("structuredheadings", {
+    currentPreset: "1.1.1.1.1.",
```

Comment:
This is the numbering scheme set on the current document, which determines which scheme new headings take, and should match the pattern of the existing headings